source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,141 @@
+Ember Addons are installed using NPM (eg. `npm install --save-dev ember-cli-sass`).
+
+Addons can automatically add dependencies to your Project's `bower.json` file.
+
+## Compiling Assets
+
+When you're using Bower dependencies that are not included in an Addon, you will
+have to instruct Ember CLI to include your assets in the build. This is done using
+the Asset manifest file `ember-cli-build.js`.
+
+#### Javascript assets exporting a globals
+
+Provide the asset path as the first and only argument.
+
+```ember-cli-build.js
+app.import('bower_components/moment/moment.js');
+```
+
+The globals provided by the asset ( `moment` in the example below can now
+be used in your application without the need to `import` them. You will need to
+tell JSHint that by adding `/* global MY_GLOBAL */` to the files where you're using globals, or
+by defining them within the `predefs` section of your `.jshintrc` file.
+
+#### AMD Javascript modules
+
+Provide the asset path as the first argument, and the list of modules and exports as the second.
+
+```ember-cli-build.js
+app.import('bower_components/ic-ajax/dist/named-amd/main.js', {
+  exports: {
+    'ic-ajax': [
+      'default',
+      'defineFixture',
+      'lookupFixture',
+      'raw',
+      'request',
+    ]
+  }
+});
+```
+
+You can now `import` them in your app. Eg. `import { raw as icAjaxRaw } from 'ic-ajax';`
+
+#### Environment Specific Assets
+
+If you need to use different assets in different environments, specify an object as the first parameter. That object's key should be the environment name, and the value should be the asset to use in that environment.
+
+```ember-cli-build.js
+app.import({
+  development: 'bower_components/ember/ember.js',
+  production:  'bower_components/ember/ember.prod.js'
+});
+```
+
+If you need to import an asset in one environment but not import it or any
+alternatives in other environments then you can wrap `app.import` in an `if` statement.
+For assets needed during testing, you should also use the `{type: 'test'}` option to make sure they
+are available in test mode.
+
+```ember-cli-build.js
+if (app.env === 'development') {
+  app.import('vendor/ember-renderspeed/ember-renderspeed.js');
+}
+if (app.env === 'test') {
+  app.import( app.bowerDirectory + '/sinonjs/sinon.js', { type: 'test' } );
+  app.import( app.bowerDirectory + '/sinon-qunit/lib/sinon-qunit.js', { type: 'test' } );
+}
+```
+
+#### Static CSS
+
+Provide the asset path as the first argument:
+
+```ember-cli-build.js
+app.import('bower_components/foundation/css/foundation.css');
+```
+
+All style assets added this way will be concatenated and output as `/assets/vendor.css`.
+
+#### Dynamic Styles (SCSS, LESS, etc)
+
+The vendor trees that are provided upon instantiation are available to your dynamic style files.  Take the following example (in `app/styles/app.scss`):
+
+```scss
+@import "bower_components/foundation/scss/normalize.scss";
+```
+
+#### Other Assets
```

Comment:
is `app.import` documented in the API somewhere? I'm wondering if we can just link there and move on